src/validation_workflow/docker/validation_docker.py

```diff
@@ -87,43 +130,50 @@ def validation(self) -> bool:
                     logging.info(f'All tests Pass : {_counter}')
                     return True
                 else:
+                    logging.info(f'Not all tests Pass : {_counter}')
+                    self.cleanup()
                     raise Exception(f'Not all tests Pass : {_counter}')
             else:
                 raise Exception('The container failed to start. Exiting the validation.')
         else:
-            logging.info('\n\nImage digest is Not validated. Exiting..\n\n')
-            raise Exception('Image digest does not match between the opensearchstaging at dockerHub/ecr and opensearchproject at local download. Exiting the validation.')
+            return True
 
     def check_http_request(self) -> bool:
         try:
-            status_code, response_text = ApiTest('https://localhost:9200/').api_get()
-            if status_code == 200:
-                return True
-            else:
-                return False
-        except (requests.exceptions.ConnectionError, requests.exceptions.ConnectTimeout) as e:
-            logging.error(f'Error connecting to https://localhost:9200/: {e}')
+            subprocess.check_output(['curl', 'https://localhost:9200', '-u', 'admin:admin', '--insecure', '-s'])
```

Comment:
Wondering if we want the `curl` to be silent "-s". Let it be loud. Even if it fails exact error can be viewed then.

Reply:
sure, @gaiksaya .. I wanted to slience it because there was too much noises, informations and caused the log/output difficult to read, but I can certainly remove the "-s" ...